tox.ini

```diff
@@ -13,120 +13,24 @@ skipsdist = true
 setenv =
     QT_QPA_PLATFORM_PLUGIN_PATH={envdir}/Lib/site-packages/PyQt5/plugins/platforms
     PYTEST_QT_API=pyqt5
+    pyqt{,56,571,58,59}: QUTE_BDD_WEBENGINE=true
+    cov: PYTEST_ADDOPTS=--cov --cov-report xml --cov-report=html --cov-report=
 passenv = PYTHON DISPLAY XAUTHORITY HOME USERNAME USER CI TRAVIS XDG_* QUTE_* DOCKER
+basepython =
+    py35: python3.5
+    py36: python3.6
 deps =
     -r{toxinidir}/requirements.txt
     -r{toxinidir}/misc/requirements/requirements-tests.txt
+    pyqt: PyQt5
+    pyqt56: PyQt5==5.6
```

Comment:
Qt 5.6 is unsupported by qutebrowser, so I think it doesn't make much sense to include it here. My bad though, since I apparently never removed the `-pyqt56` envs 😆